File: _includes/code/fibonacci.cpp
#include <iostream>
#include <cstdlib>

int fibonacci(int i) {
	if (i <= 1) {
		return i;
	} else {
		return fibonacci(i-1) + fibonacci(i-2);
	}
}

int main(int argc, char* argv[]) {
	if (argc < 2) {
		std::cerr << "Usage: " << argv[0] << " number" << std::endl;
	} else {
		int n = atoi(argv[1]);
		std::cout << "F(" << n << ") = " << fibonacci(n) << std::endl;
	}
	return 0;
}

File: _includes/code/fibonacci.pl
#!/usr/bin/perl
use strict;

sub fibonacci {
	my $i = shift;
	return $i if $i <= 1;
	return fibonacci($i-1) + fibonacci($i-2) if $i > 1;
}

if (@ARGV < 1) {
	print STDERR "Usage: $0 number\n";
} else {
	my $n = shift @ARGV;
	print "F($n) = ", fibonacci($n), "\n";
}
